dbt_platform_helper/utils/git.py

```diff
@@ -10,6 +10,7 @@ def git_remote():
     if not git_repo:
         return
 
-    _, repo = git_repo.split("@")[1].split(":")
+    # Temporary hacked in line from Ben's branch to enable work in this branch to continue
+    repo = re.search(r"[^/:]*?/[^/]*?\.git\Z", git_repo).group()
```

Comment:
We will get @benn-dbt's branch merged and merge `main` into here before merging this one.

Reply:
@WillGibson @benn-dbt will this be the final solution in the end, as opposed to the slightly more readable alternative that was being discussed in Slack?